pkg/mesh/app.go

```diff
@@ -38,6 +38,8 @@ const (
 
 // StartApp uses the reminder of the command line to exec an app, using K8S_UID as UID, if present.
 func (kr *KRun) StartApp() {
+	// Even if there isn't an app to start, we still need to forward SIGTERM/SIGINT to envoy.
+	defer kr.Signals()
```

Comment:
maybe just call it explicitly before StartApp ? Not sure why 'defer', does order matter ?

Reply:
In case there is no args to krun (no customer app), there is an early return which would mean we don't call kr.Signals which is needed regardless of whether a customer app is present. We could also revert that and selectively call Startup based on an env variable, but this may be cleaner, up to you.